docs/reference/setup/important-settings/discovery-settings.asciidoc

```diff
@@ -1,58 +1,56 @@
 [[discovery-settings]]
-=== Discovery settings
+=== Discovery and cluster formation settings
 
-Elasticsearch uses a custom discovery implementation called "Zen Discovery" for
-node-to-node clustering and master election. There are two important discovery
-settings that should be configured before going to production.
+There are two important discovery and cluster formation settings that should be
+configured before going to production so that nodes in the cluster can discover
+each other and elect a master node.
 
 [float]
 [[unicast.hosts]]
 ==== `discovery.zen.ping.unicast.hosts`
 
 Out of the box, without any network configuration, Elasticsearch will bind to
-the available loopback addresses and will scan ports 9300 to 9305 to try to
-connect to other nodes running on the same server. This provides an auto-
+the available loopback addresses and will scan local ports 9300 to 9305 to try
+to connect to other nodes running on the same server. This provides an auto-
 clustering experience without having to do any configuration.
 
-When the moment comes to form a cluster with nodes on other servers, you have to
-provide a seed list of other nodes in the cluster that are likely to be live and
-contactable. This can be specified as follows:
+When the moment comes to form a cluster with nodes on other servers, you must
+use the `discovery.zen.ping.unicast.hosts` setting to provide a seed list of
+other nodes in the cluster that are master-eligible and likely to be live and
+contactable.  This setting should normally contain the addresses of all the
+master-eligible nodes in the cluster.
+
```

Comment:
I would suggest moving the reference details about this setting here.  For example:

```suggestion
This setting contains either an array of hosts or a comma-delimited string. Each
value should be in the form of `host:port` or `host` (where `port` defaults to
the setting `transport.profiles.default.port` falling back to `transport.port`
if not set). Note that IPv6 hosts must be bracketed. The default for this
setting is `127.0.0.1, [::1]`
```